chef/roles/logging-client.rb

```diff
@@ -2,7 +2,7 @@
 name "logging-client"
 description "Logging Client Role - Logging client for the cloud points to Master"
 run_list(
-         "recipe[logging::client]"
+         "recipe[logging::role_logging_client]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.